src/plugins/intel_gna/tests/functional/shared_tests_instances/subgraph_tests/convolution_relu_sequence.cpp

```diff
@@ -236,34 +240,46 @@ const std::vector<convReluSpecificParams> poolingStrideAboveWindow = {
     },
 };
 
-const std::vector<convReluSpecificParamsAll> poolingStrideNotEqualWindowAll = {
-    {inputShape1Even, poolingStrideBelowWindow},
-    {inputShape1DOneAbove, poolingStrideBelowWindow},
-    {inputShape1DOneBelow, poolingStrideBelowWindow},
+const std::vector<convReluSpecificParamsAll> poolingStrideNotEqualWindowAll_Above = {
     {inputShape1Even, poolingStrideAboveWindow},
     {inputShape1DOneAbove, poolingStrideAboveWindow},
     {inputShape1DOneBelow, poolingStrideAboveWindow},
-    {inputShape1DMultichannel4, poolingStrideBelowWindow},
     {inputShape1DMultichannel4, poolingStrideAboveWindow},
-    {inputShape1DMultichannel5, poolingStrideBelowWindow},
     {inputShape1DMultichannel5, poolingStrideAboveWindow},
-    {inputShape1DMultichannel6, poolingStrideBelowWindow},
     {inputShape1DMultichannel6, poolingStrideAboveWindow},
-    {inputShape1DMultichannel7, poolingStrideBelowWindow},
     {inputShape1DMultichannel7, poolingStrideAboveWindow},
-    {inputShape1DMultichannel8, poolingStrideBelowWindow},
     {inputShape1DMultichannel8, poolingStrideAboveWindow},
-    {inputShape1DMultichannel9, poolingStrideBelowWindow},
     {inputShape1DMultichannel9, poolingStrideAboveWindow}};
 
-INSTANTIATE_TEST_SUITE_P(smoke_ConvolutionPoolingStrideNotEqualWindowTest,
+INSTANTIATE_TEST_SUITE_P(smoke_ConvolutionPoolingStrideNotEqualWindowTest_Above,
                          ConvolutionReluSequenceTest,
-                         ::testing::Combine(::testing::ValuesIn(poolingStrideNotEqualWindowAll),
+                         ::testing::Combine(::testing::ValuesIn(poolingStrideNotEqualWindowAll_Above),
                                             ::testing::ValuesIn(netPrecisions),
                                             ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
                                             ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
                                             ::testing::Values(CommonTestUtils::DEVICE_GNA),
-                                            ::testing::ValuesIn(configs)),
+                                            ::testing::ValuesIn(configs_exec_target_2_0)),
+                         ConvolutionReluSequenceTest::getTestCaseName);
+
+const std::vector<convReluSpecificParamsAll> poolingStrideNotEqualWindowAll_Below = {
+    {inputShape1Even, poolingStrideBelowWindow},
+    {inputShape1DOneAbove, poolingStrideBelowWindow},
+    {inputShape1DOneBelow, poolingStrideBelowWindow},
+    {inputShape1DMultichannel4, poolingStrideBelowWindow},
+    {inputShape1DMultichannel5, poolingStrideBelowWindow},
+    {inputShape1DMultichannel6, poolingStrideBelowWindow},
+    {inputShape1DMultichannel7, poolingStrideBelowWindow},
+    {inputShape1DMultichannel8, poolingStrideBelowWindow},
+    {inputShape1DMultichannel9, poolingStrideBelowWindow}};
 
+// GNA 3_0 and above check if pooling stride is less or equal to window so we cannot use value which is above.
```

Comment:
Are you sure pooling stride does not have to be <= pooling window for GNA 2.0 as well?

Reply:
It looks like it is not limited in the same way for older versions of GNA.